wgengine/magicsock/magicsock.go

```diff
@@ -4455,8 +4471,9 @@ func (de *endpoint) send(buffs [][]byte) error {
 		return errNoUDPOrDERP
 	}
 	var err error
+
 	if udpAddr.IsValid() {
-		_, err = de.c.sendUDPBatch(udpAddr, buffs)
+		_, err := de.c.sendUDPBatch(udpAddr, buffs)
```

Comment:
this looks wrong, it was previously updating the err outside of this block

Reply:
:O good catch, thank you.